Resources/Private/Partials/FrontendAPA.html

```diff
@@ -4,6 +4,7 @@
         xmlns:n="http://typo3.org/ns/UMA/UmaPublist/ViewHelpers">
 
 <v:variable.set name="publication.title" value="{publication.title->v:format.trim()}" />
+<v:variable.set name="publication.pageRange" value="{v:format.replace(substring: '-', content: {publication.pageRange}, replacement: '–')}" />	
```

Comment:
That produced an error because the variable in parameter `content` should be entered without the curly braces!